pyfstat/grid_based_searches.py

```diff
@@ -101,14 +101,13 @@ def __init__(
             os.mkdir(outdir)
         self.set_out_file()
         self.search_keys = ["F0", "F1", "F2", "Alpha", "Delta"]
-        self.input_keys = ["minStartTime", "maxStartTime"] + self.search_keys
-        self.keys = self.input_keys.copy()
+        self.output_keys = self.search_keys.copy()
```

Comment:
And `keys` seemed too ambiguous, what it actually already was was the list for the output `data` array, so I renamed it appropriately.